sys/net/include/etx_beaconing.h

```diff
@@ -26,43 +26,45 @@
 extern "C" {
 #endif
 
-//[option|length|ipaddr.|packetcount] with up to 15 ipaddr|packetcount pairs
-// 1 Byte 1 Byte  1 Byte  1 Byte
+/** 
+ * @brief [option|length|ipaddr.|packetcount] with up to 15 ipaddr|packetcount pairs
+ *         1 Byte 1 Byte  1 Byte  1 Byte 
+ */
```

Comment:
I have not looked at the generated documentation, but does this comment really look okay in the generated docs?

If not, try something like http://www.stack.nl/~dimitri/doxygen/manual/markdown.html#md_tables